examples/browser-webrtc/README.md

```diff
@@ -0,0 +1,22 @@
+# Rust-Libp2p Browser-Server WebRTC Example
+
+This example demonstrates how to use the `rust-libp2p` library in a browser.
+It uses [Leptos](https://leptos.dev/) WebAssembly framework to display the pings from the server.

```

Comment:
Same here. Is `leptos` a requirement? I would like to keep the dependency footprint low. Especially given the maintenance overhead it implies.

I am assuming that it gives us a nice front-end framework. Though is that important for showcasing the transport?

Reply:
Fair points, yes it is a front end framework not necessary to chowcase the Transport. I am removing `leptos` and switching to vanilla `wasm-bindgen` with a `wasm-pack` build, it's nearly complete now.